drivers/staging/android/ion/Kconfig

```diff
@@ -52,3 +52,39 @@ config ION_OF
 	  extensions
 
 	  If using Ion and devicetree, you should say Y here
+
+config ION_UNMAPED
+	bool "Driver for an Ion heap type using unmapped heap"
+	depends on ION
+	help
+	  Provides a ION heap type for handling memory pool that are
+	  default not mapped to the linux kernel. This can be used
+	  when linux executes in a non secure state and the target
+	  memory references are expected to be accesed from secure
+	  agent only.
+	  Ion "unmaped" heap rely on Ion UAPI to identify itself.
+	  ION_HEAP_TYPE_UNMAPED = ION_HEAP_TYPE_CUSTOM + 1
```

Comment:
I intentionally did not change the ION UAPI, since it is UAPI. I preferred to insure "UNMAP**P**ED" heap type ID rely on already defined UAPI IDs, hence  CUSTOM + _something_, both in kernel and user lands.